fairlearn/reductions/_moments/moment.py

```diff
@@ -33,20 +33,61 @@ def load_data(self, X, y, **kwargs):
         assert self.data_loaded is False, \
             "data can be loaded only once"
         self.X = X
-        self.n = self.X.shape[0]
         self.tags = pd.DataFrame({_LABEL: y})
         if _KW_SENSITIVE_FEATURES in kwargs:
             self.tags[_GROUP_ID] = kwargs[_KW_SENSITIVE_FEATURES]
         self.data_loaded = True
         self._gamma_descr = None
 
+    @property
+    def X(self):
+        """Return the array of features."""
+        return self._X
+
+    @X.setter
+    def X(self, value):
+        self._X = value
+
+    @property
+    def n(self):
```

Comment:
maybe this is a good time to rename this from `n` to something more descriptive like `n_samples`

Reply:
Agreed and done.